sdk/schemaregistry/azure-schemaregistry-client/src/main/java/com/azure/schemaregistry/client/rest/AzureSchemaRegistryRestService.java

```diff
@@ -0,0 +1,1411 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.schemaregistry.client.rest;
```

Comment:
This should not be used directly by the user. So, move this to `implementation` package. Same for any other classes that shouldn't in public API view.